Frameworks/CoreText/CTFontDescriptor.mm

```diff
@@ -15,59 +15,124 @@
-const CFStringRef kCTFontSymbolicTrait = static_cast<CFStringRef>(@"kCTFontSymbolicTrait");
-const CFStringRef kCTFontWeightTrait = static_cast<CFStringRef>(@"kCTFontWeightTrait");
-const CFStringRef kCTFontWidthTrait = static_cast<CFStringRef>(@"kCTFontWidthTrait");
-const CFStringRef kCTFontSlantTrait = static_cast<CFStringRef>(@"kCTFontSlantTrait");
+const CFStringRef kCTFontURLAttribute = static_cast<CFStringRef>(@"NSFontURLAttribute");
+const CFStringRef kCTFontNameAttribute = static_cast<CFStringRef>(@"NSFontNameAttribute");
+const CFStringRef kCTFontDisplayNameAttribute = static_cast<CFStringRef>(@"NSFontVisibleNameAttribute");
+const CFStringRef kCTFontFamilyNameAttribute = static_cast<CFStringRef>(@"NSFontFamilyAttribute");
+const CFStringRef kCTFontStyleNameAttribute = static_cast<CFStringRef>(@"NSFontFaceAttribute");
+const CFStringRef kCTFontTraitsAttribute = static_cast<CFStringRef>(@"NSFontTraitsAttribute");
+const CFStringRef kCTFontVariationAttribute = static_cast<CFStringRef>(@"NSFontVariationAttribute");
+const CFStringRef kCTFontSizeAttribute = static_cast<CFStringRef>(@"NSFontSizeAttribute");
+const CFStringRef kCTFontMatrixAttribute = static_cast<CFStringRef>(@"NSFontMatrixAttribute");
+const CFStringRef kCTFontCascadeListAttribute = static_cast<CFStringRef>(@"NSFontCascadeListAttribute");
+const CFStringRef kCTFontCharacterSetAttribute = static_cast<CFStringRef>(@"NSFontCharacterSetAttribute");
+const CFStringRef kCTFontLanguagesAttribute = static_cast<CFStringRef>(@"NSFontLanguagesAttribute");
+const CFStringRef kCTFontBaselineAdjustAttribute = static_cast<CFStringRef>(@"NSFontBaselineAdjustAttribute");
+const CFStringRef kCTFontMacintoshEncodingsAttribute = static_cast<CFStringRef>(@"NSFontMacintoshEncodingsAttribute");
+const CFStringRef kCTFontFeaturesAttribute = static_cast<CFStringRef>(@"NSFontFeaturesAttribute");
+const CFStringRef kCTFontFeatureSettingsAttribute = static_cast<CFStringRef>(@"NSFontFeatureSettingsAttribute");
+const CFStringRef kCTFontFixedAdvanceAttribute = static_cast<CFStringRef>(@"NSFontFixedAdvanceAttribute");
+const CFStringRef kCTFontOrientationAttribute = static_cast<CFStringRef>(@"NSFontOrientationAttribute");
+const CFStringRef kCTFontFormatAttribute = static_cast<CFStringRef>(@"NSFontFormatAttribute");
+const CFStringRef kCTFontRegistrationScopeAttribute = static_cast<CFStringRef>(@"NSFontRegistrationScopeAttribute");
+const CFStringRef kCTFontPriorityAttribute = static_cast<CFStringRef>(@"NSFontPriorityAttribute");
+const CFStringRef kCTFontEnabledAttribute = static_cast<CFStringRef>(@"NSFontEnabledAttribute");
+
+const CFStringRef kCTFontSymbolicTrait = static_cast<CFStringRef>(@"NSCTFontSymbolicTrait");
+const CFStringRef kCTFontWeightTrait = static_cast<CFStringRef>(@"NSCTFontWeightTrait");
+const CFStringRef kCTFontWidthTrait = static_cast<CFStringRef>(@"NSCTFontWidthTrait");
+const CFStringRef kCTFontSlantTrait = static_cast<CFStringRef>(@"NSCTFontSlantTrait");
```

Comment:
I remember there being some weird reason why we couldn't do this (ran into it in other files too) but I don't remember what it was...

---
In reply to: [79032599](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r79032599) [](ancestors = 79032599)